submission-panels.md

```diff
@@ -0,0 +1,69 @@
+# Submission-panels Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/submission-definitions**   
+
+Provide access to the configured submission definitions. It returns the list of existent submission-definitions.
+
+Example: to be provided
+
+## Single Submission-panels 
+**/api/configuration/submission-definitions/<:definition-name>**
+
+Provide detailed information about a specific submission-definition. The JSON response document is as follow
+```json
+{
+  "name": "traditional",
+  "panels": [
+  {
+  	header: {"en": "First page"},
+  	mandatory: true,
+  	type: "inputform",
+  	scope: null
+  },
+  {
+  	header: {"en": "Second page"},
+  	mandatory: true,
+  	type: "inputform",
+  	scope: null
+  },
+  {
+  	header: {"en": "Files and access condition"},
+  	mandatory: true,
+  	type: "uploadWithEmbargo",
+  	scope: null
+  },
+  {
+  	header: {"en": "Deposit license"},
+  	mandatory: true,
+  	type: "license",
+  	scope: null
+  },
+  {
+  	header: {"en": "Creative Commons"},
+  	mandatory: false,
+  	type: "cclicense",
+  	scope: null
+  },
+  ...  
+  ],
+  "isDefault": true
+}
+
+```
+
+Exposed links:
+* collections: list of collections that explicitly use such submission-definition
+
+## Search methods
+### findByCollection
+**/api/configuration/submission-definitions/search/findByCollection?uuid=<:collection-uuid>**
```

Comment:
👍 I was missing this in the first commit. But shouldn't we rename the parameter from `uuid` to `scope` to be consistent with the other endpoints?

Reply:
I don't think so. Here we offer a lookup for collection, so we need an uuid. In the other endpoints (we are talking about the public search right?) scope is broader concept that include collection and community right now but hopeful, as I propose, generic "named configuration" in future

Reply:
`scope` is also used in the Browse endpoint: https://github.com/DSpace/Rest7Contract/blob/master/browses.md#metadata-browse-1st-level

Reply:
yes but also the browse system could be in future "constraint" to a broader scope concept than community or collection. For instance in DSpace-CRIS we allow to set "virtual scope" as SOLR filter query so that one can constraint for example the browses (authors, title, keywords, etc.) to only the item where dc.type = thesis

Again here we can only accept an uuid as we are looking to the submission definition by collection

Reply:
Ok, I understand.